### PR TITLE
RA-1613:Change hover message for 'x' icon in manage conditions

### PR DESCRIPTION
**Pull request by haripriya999**
# RA-1613:Change hover message for 'x' icon in manage conditions

## Description of what I changed
changed hover message for 'x' icon in manage conditions 

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1613

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: omod/src/main/webapp/pages/conditionlist/manageConditions.gsp [hunk 1]
```diff
@@ -86,7 +86,7 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                                ng-click="activateCondition(condition)" ng-if="condition.status==='INACTIVE' && condition.voided===false"></i>
                             <i class="icon-minus-sign edit-action" title="${ui.message("coreapps.conditionui.inactive")}"
                                ng-click="deactivateCondition(condition)" ng-if="condition.status==='ACTIVE' && condition.voided===false"></i>
-                            <i class="icon-remove delete-action" title="${ui.message("coreapps.coreapps.delete")}"
+                            <i class="icon-remove delete-action" title="${ui.message("coreapps.delete")}"
```
Comment (HerbertYiga) [inline omod/src/main/webapp/pages/conditionlist/manageConditions.gsp]: does coreapps.delete arleady exist in the messages.properties file?

Comment (HerbertYiga) [reply omod/src/main/webapp/pages/conditionlist/manageConditions.gsp]: Some failing checks here,did this pass locally on building?

Comment (haripriya999) [reply omod/src/main/webapp/pages/conditionlist/manageConditions.gsp]: @HerbertYiga Yes, coreapps.delete already exists in messages.propertis file

**Comment by haripriya999** [comment]
@dkayiwa and @HerbertYiga I fixed the Travis build error. Thank you :)

**Comment by haripriya999** [comment]
@dkayiwa  Could you please take a look at this. I fixed the merge conflicts. Thank you.

**Comment by dkayiwa** [comment]
@haripriya999 the only change i see in this pull request is formatting.

**Comment by haripriya999** [comment]
@dkayiwa  Seems like I reverted the changes while resolving the merge conflicts. I updated the PR.

**Pull request merged**